### ISSUE TITLE
Forge makes unexpected modifications to fsproj files

### ISSUE THREAD

**Issue by alfonsogarciacaro** [ionide/ionide-vscode-fsharp#82]
Not sure if this should be reported here, but Ionide is making unexpected changes to my `fsproj` files so I have to carefully check before committing anything to discard these changes. This happens without using any particular command to alter the `fsproj` file (like "Add file", etc...), just opening files belonging to the project. Some of these changes are just reordering properties and post no harm, but others look like errors. For example:

- Some `dll` references are removed, not sure what the pattern is.
- Some properties like `StartArguments` or `StartWorkingDirectory` are removed.
- `DefineConstants` removes semicolons:

```diff
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>DEBUGTRACE</DefineConstants>
```

I can try to reproduce the problem and push it in a commit for illustration if you need it.

Cheers!

**Comment by Krzysztof-Cieslak** [comment]
Reproduction for removing `dll` references would be helpful. Rest of problems should be easy to fix.

**Comment by alfonsogarciacaro** [comment]
I've reproduced the problem and made a test commit [here](https://github.com/fsprojects/Fable/commit/6a22d3b5791cd78647dfd98179c3369f9854ad32). You can see that the reference to `NUnit.dll` disappears. Please note that there is a `paket.references` in the project folder including NUnit.

Also, the information I gave above was not quite right: `fsproj` is not updated just by opening a file belonging to the project, but in this case it happened when changing the name of one of the files.

I hope this helps!

**Comment by cloudRoutine** [comment]
@alfonsogarciacaro could you report this on https://github.com/fsprojects/Forge/issues ?

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `2.0.0`, let me know if there are still not correct modifications. (also, automatic modification of project files on add/remove file is turned off by default, there is option enabling it)

**Comment by alfonsogarciacaro** [comment]
Great, thanks! I'll try :+1: Also, it's nice that project file modification is an opt-in, I had to discard Forge modifications many times in my commits 😉
